### PR TITLE
Misc edits to getting started section

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2-docs/issues/472.

### CODE REVIEW

File: src/app/learn/get-started/advanced/add-entry-components/index.html [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-  <p>
-    After you create a component in the default single-page application, you can create another page and add a SKY UX modal component to register an entry component.
+    The following tutorial walks you through the process to create a page with a SKY UX modal component and declare that component as an entry component.
   </p>
-
   <ol>
     <li>
       <p>
-        Return to the project in a local editor, and create a folder within the <stache-code>src/app</stache-code> folder.
+        Open your project in a local editor, and create a folder within the <stache-code>src/app</stache-code> folder. The folder name appears in the route for the page. For this tutorial, use <stache-code>modal</stache-code> for the folder name.
```
Comment (Blackbaud-AlexKingman) [inline src/app/learn/get-started/advanced/add-entry-components/index.html]: > in the route for the page
What does "the route" refer to here? I'm not quite sure I follow this sentences intent.

Comment (blackbaud-johnly) [reply src/app/learn/get-started/advanced/add-entry-components/index.html]: I updated the second sentence to "The folder name is represented in the page's URL."

File: src/app/learn/get-started/advanced/add-entry-components/index.html [hunk 2]
```diff
@@ -1,32 +1,32 @@
 <stache
-  pageTitle="Add Entry Components"
+  pageTitle="Entry Components"
```
Comment (Blackbaud-BobbyEarl) [inline src/app/learn/get-started/advanced/add-entry-components/index.html]: Would you want to also rename this folder to `entry-components` and add a redirect from `add-entry-components` to `entry-components`?

Comment (blackbaud-johnly) [reply src/app/learn/get-started/advanced/add-entry-components/index.html]: I renamed the folder and added a redirect.

File: src/app/learn/get-started/advanced/entry-components/index.html [hunk 3]
```diff
@@ -175,7 +170,7 @@ <h1>Modal launch page</h1>
     </li>
     <li>
       <p>
-        In the <stache-code>modal</stache-code> folder, add another new file and name it <stache-code>modal-demo-launcher.component.html</stache-code>. This file serves as the template for the button component that launches the modal form. Add the following code:
+        In the <stache-code>modal</stache-code> folder, create a <stache-code>modal-demo-launcher.component.html</stache-code file to serve as the template for the button component that launches the modal form and add the following code:
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/learn/get-started/advanced/entry-components/index.html]: missing the > right before "file to serve"

Comment (blackbaud-johnly) [reply src/app/learn/get-started/advanced/entry-components/index.html]: Nice catch. I'm surprised that didn't throw an error when I served. ... Fixed.

**Pull request merged**